### ISSUE TITLE
Support React 16

### ISSUE THREAD

**Issue by lostthetrail** [bsidelinger912/shiitake#15]
```
npm WARN shiitake@2.1.1 requires a peer of react@^15.5.4 but none is installed. You must install peer dependencies yourself.
```

**Comment by bsidelinger912** [comment]
I just published v2.2.1 and tested using yarn with React 16 (and 15), and it looks good to me.  Please let me know when you have a chance to try it.

**Comment by bsidelinger912** [comment]
Closing based on confirmation here: https://github.com/bsidelinger912/shiitake/pull/16
